plugins/linuxv2/model/namespace/namespace.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package namespace;
+
+message Namespace {              /* Linux network namespace */
+    enum NamespaceType {
+        PID_REF_NS = 0;          /* namespace of a given process */
```

Comment:
Since we are doing rewrite of the API.. we should consider defining these network namespace types same way as they are [defined in the linux kernel](https://github.com/torvalds/linux/blob/6f0d349d922ba44e4348a17a78ea51b7135965b1/include/uapi/linux/net_namespace.h#L13-L19).

```c
enum {
	NETNSA_NONE,
	NETNSA_NSID,
	NETNSA_PID,
	NETNSA_FD,
    //...
};
```

Reply:
OK, but will also add our own NETNSA_MICROSERVICE

Reply:
Just to clarify.. I did not mean to suggest to use `NETNSA_` prefix. My intention was to follow the same order, basically have same value for same type.